### PR TITLE
[php] Enable onboarding tests

### PR DESCRIPTION
**Pull request by iamluc**
## Changes

Enable `INSTALLER_AUTO_INJECTION` tests for PHP

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: docs/execute/requirements.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ You''ll need a bash based terminal, docker and python3.12
 ```
 sudo add-apt-repository ppa:deadsnakes/ppa
 sudo apt update
-sudo apt install python3.12 python3.12-distutils python3.12-venv
+sudo apt install python3.12 python3.12-distutils python3.12-venv python3.12-dev
```
Comment (robertomonteromiguel) [inline docs/execute/requirements.md:10]: why this change?

**Review by robertomonteromiguel — APPROVED** [review]

File: docs/execute/requirements.md (hunk 1 above)
Comment (iamluc) [reply docs/execute/requirements.md:10]: IIRC, I had issue while running `./build.sh`.
`pip` was not able to install some packages

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
I can see on gitlab errors related with the dockerhub rate limits.
```#2 ERROR: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/library/php/manifests/sha256:d5338bd62184c4ae4526b9e7a93de80acc5436e31d262f3063a4ad5ed4b1308a: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
------
 > [internal] load metadata for docker.io/library/php:8.3:
------
Dockerfile:1
--------------------
   1 | >>> FROM php:8.3
   2 |     
   3 |     RUN mkdir /app
```
Recently we changes all references from dockerhub to public.ecr.aws. Could you try to use aws registry too?
java example: https://github.com/DataDog/system-tests/blob/main/utils/build/virtual_machine/weblogs/java/test-app-java-container/Dockerfile.jdk11

**Comment by iamluc** [comment]
> Could you try to use aws registry too?

Done, and tests pass now! See https://gitlab.ddbuild.io/DataDog/system-tests/-/pipelines/39380671

**Review by robertomonteromiguel — APPROVED** [review]
